codegenerator/cli/templates/static/codegen/src/eventFetching/SourceManager.res

```diff
@@ -6,27 +6,36 @@ open Belt
 // So this module is to encapsulate the fetching logic only
 // with a mutable state for easier reasoning and testing.
 type t = {
-  logger: Pino.t,
   sources: array<Source.t>,
   maxPartitionConcurrency: int,
+  newBlockFallbackStallTimeout: int,
```

Comment:
I love explicit naming 😄

Reply:
Although I prefer to add the unit as well if it's just an int. Like, millis/ms or sec

Reply:
I don't know. This is going to be too long and milliseconds are the default for configs like this, so I think it's fine.